src/operator/LimitsField.tsx

```diff
@@ -36,12 +40,35 @@ const LimitsField: FC<Props> = ({type, name, limits, onChangeLimits}) => {
     onChangeLimits(newLimits)
   }
 
+  if (type == InputType.Number) {
+    return hasFocus ? (
+      <Input
+        type={type}
+        name={name}
+        value={value}
```

Comment:
shouldn't this also be `csv` (or whatever you change the name to, if you change it) since the code is already checking the type above?

Reply:
In this case it needs to be the value, since a number input can't have commas in it. I'm swapping the input to a text input `onBlur` so they can see the value with commas (if large enough).